backend/src/main/resources/graphql/main.graphqls

```diff
@@ -354,7 +367,8 @@ type Mutation {
         regexp: "[0-9]{5}(-[0-9]{4})?"
         message: "Use ZIP or ZIP+4 format"
       )
-    telephone: String!
+    telephone: String
+    phoneNumbers: [PhoneNumberInput!]
```

Comment:
is there any reason not to do `[PhoneNumberInput!]!`

Reply:
At least temporarily, we'd like to remain backwards-compatible with older UI versions that won't send `phoneNumbers` in mutations. In that case, the service would construct `PhoneNumberInput` from the provided `telephone`:

https://github.com/CDCgov/prime-simplereport/blob/b449fde556b64f392ccbd272112cca837af90a17/backend/src/main/java/gov/cdc/usds/simplereport/api/patient/PatientMutationResolver.java#L82-L85